ui/v2.5/src/components/Shared/GridCard/GridCard.tsx

```diff
@@ -78,6 +79,31 @@ export const useContainerDimensions = <T extends HTMLElement = HTMLDivElement>(
   return [target, dimension];
 };
 
+export function useCardWidth(
+  containerWidth: number,
+  zoomIndex: number,
+  zoomWidths: number[]
+) {
+  return useMemo(() => {
```

Comment:
Thanks for this update. I'll need to remember that `useMemo` can be used like this to minimize operations.